test/extensions/filters/listener/tls_inspector/tls_inspector_fuzz_test.cc

```diff
@@ -0,0 +1,41 @@
+#include "extensions/filters/listener/tls_inspector/tls_inspector.h"
+
+#include "test/extensions/filters/listener/common/fuzz/listener_filter_fuzzer.h"
+#include "test/extensions/filters/listener/tls_inspector/tls_inspector_fuzz_test.pb.validate.h"
+#include "test/fuzz/fuzz_runner.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace ListenerFilters {
+namespace TlsInspector {
+
+DEFINE_PROTO_FUZZER(
+    const test::extensions::filters::listener::tls_inspector::TlsInspectorTestCase& input) {
+
```

Comment:
If you need to do another push, tiny nit would be to task to remove this superfluous blank line.